_source/_posts/2022-06-01-jhipster-digital-ocean.md

```diff
@@ -0,0 +1,411 @@
+---
+layout: blog_post
+title: "JHipster Application Deployment to DigitalOcean's Kubernetes"
+author: jimena-garbarino
+by: contractor
+communities: [devops,java]
+description: "A step-by-step guide for JHipster deployment to DigitalOcean's cloud"
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
+---
+
+Cloud adoption continues to increase rapidly and worldwide, and not only in the software industry. Every year more and more companies move their applications to the cloud. In the last JHipster community survey, in December 2021, users who participated expressed they value getting to production fast, thanks to JHipster, and requested more tutorials on deployment to cloud platforms. DigitalOcean is among the most popular "other" cloud vendors, according to some surveys. This post is a quick walk-through of the deployment of a JHipster microservices architecture to a Kubernetes cluster in DigitalOcean's cloud.
+
+**This tutorial was created with the following frameworks and tools**:
+- [JHipster 7.6.0](https://www.jhipster.tech/installation/)
+- [Java OpenJDK 11](https://jdk.java.net/java-se-ri/11)
```

Comment:
FYI... it doesn't work with Java 17. Not sure if we should mention that.

```
Execution failed for task ':compileJava'.
> java.lang.IllegalAccessError: class org.gradle.internal.compiler.java.ClassNameCollector (in unnamed module
 @0x33ec151d) cannot access class com.sun.tools.javac.code.Symbol$TypeSymbol (in module jdk.compiler) 
because module jdk.compiler does not export com.sun.tools.javac.code to unnamed module @0x33ec151d
```

Reply:
Added a comment about that.